dao/src/main/scala/META-INF/MANIFEST.MF

```diff
@@ -1,3 +1,3 @@
-Manifest-Version: 1.0
-Class-Path: 
-
+Manifest-Version: 1.0
```

Comment:
Let's remove this file

Reply:
This is just a CRLF->LF switch, a possible removal can be done separately.